app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                              'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
1. Coding convention:
+ Trong Ruby sử dụng nháy đôi nhé
chỉ trong js, css file thì sử dụng nháy đơn thôi
+ Các thành phần tách nhau 1 space thôi nhé - sau `stylesheet_link_tag` 1 space

2. `Ruby on Rails Tutorial Sample App`
Tìm hiểu `I18n in rails` rồi sử dụng cho text của toàn bộ các file `.html.erb` nhé em